.changeset/eighty-dolphins-approve.md

```diff
@@ -0,0 +1,5 @@
+---
+"@jspsych-contrib/plugin-html-vas-response": patch
```

Comment:
This should be a `minor` version instead of a `patch`, since you are adding a new feature